### PR TITLE
Update Doc Gen swagger definition, update endpoint address in config

### PR DESCRIPTION
**Pull request by emiaj**

### CODE REVIEW

File: samples/Kmd.Logic.DocumentGeneration.Client.ConfigurationSample/appsettings.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   },
   "DocumentGeneration": {
     "SubscriptionId": "00000000-0000-0000-0000-000000000000",
-    "DocumentGenerationServiceUri": "https://gateway.kmdlogic.io/document-generation/v2"
```
Comment (danielhazel) [inline samples/Kmd.Logic.DocumentGeneration.Client.ConfigurationSample/appsettings.json]: This is not the same v2.
This one is the version of the API in APIM.

File: samples/Kmd.Logic.DocumentGeneration.Client.GenerationSample/appsettings.json [hunk 2]
```diff
@@ -2,11 +2,11 @@
   "TokenProvider": {
     "ClientId": "00000000-0000-0000-0000-000000000000",
     "ClientSecret": "",
-    "AuthorizationScope": "https://logicidentityprod.onmicrosoft.com/00000000-0000-0000-0000-000000000000/.default",
+    "AuthorizationScope": "https://logicidentityprod.onmicrosoft.com/00000000-0000-0000-0000-000000000000/.default"
   },
   "DocumentGeneration": {
     "SubscriptionId": "00000000-0000-0000-0000-000000000000",
-    "DocumentGenerationServiceUri": "https://gateway.kmdlogic.io/document-generation/v2",
```
Comment (danielhazel) [inline samples/Kmd.Logic.DocumentGeneration.Client.GenerationSample/appsettings.json:9]: This is not the same v2.
This one is the version of the API in APIM.

File: samples/Kmd.Logic.DocumentGeneration.Client.ConfigurationSample/appsettings.json (hunk 1 above)
Comment (emiaj) [reply samples/Kmd.Logic.DocumentGeneration.Client.ConfigurationSample/appsettings.json]: Fixed in 01522a4eaf410099e003bcc977431c76fee743e4

File: samples/Kmd.Logic.DocumentGeneration.Client.GenerationSample/appsettings.json (hunk 2 above)
Comment (emiaj) [reply samples/Kmd.Logic.DocumentGeneration.Client.GenerationSample/appsettings.json:9]: Fixed in 01522a4eaf410099e003bcc977431c76fee743e4

**Comment by emiaj** [comment]
> I don't see a reason to have merged from master.
> :(

Oops, sorry about that, I can hard push to fix the log graph :)

**Comment by emiaj** [comment]
> > I don't see a reason to have merged from master.
> > :(
> 
> Oops, sorry about that, I can hard push to fix the log graph :)

Done, log history has been fixed (I had to `push -f`).

**Pull request merged**